app/testOnly/views/featureSwitch.scala.html

```diff
@@ -34,6 +34,7 @@ <h1 class="heading-medium">Features</h1>
                 @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.allowDirectDebitsFeature), "Direct debits allowed")
                 @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.staticDateEnabledFeature), "Use static date (2018-05-01)")
                 @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.vatCertificateFeature), "Show vat certificate")
+                @views.html.templates.inputs.singleCheckbox(form(ConfigKeys.vatOptOutServiceFeature), "Show vat opt-out service")
```

Comment:
Could you change this feature switch description to something like "Show vat opt-out section", just to be clear on what it is for others who see it.